Sources/Player/PlaybackEngine/Internal/StreamingPrivileges/StreamingPrivilegesHandler.swift

```diff
@@ -204,6 +210,7 @@ private extension StreamingPrivilegesHandler {
 		case let .string(text):
 			await interpret(text)
 		default:
+			PlayerWorld.logger?.log(loggable: PlayerLoggable.streamingHandleInvalidMessage(message: String(describing: message)))
 			break
```

Comment:
There is a warning here regarding the break as it does not seem needed anymore.

Reply:
ahh yes, no need anymore :)

Reply:
fixed ✅